### ISSUE TITLE
Issue with Example Kernels

### ISSUE THREAD

**Issue by kellyrowland** [lightsighter/Weft#10]
Hi,

I'm trying to run the Weft test kernels but am running into the same execution error. When trying to run weft on saxpy_single.ptx, I get the following:

```
$ weft -n 320 saxpy_single.ptx
weft: program.cc:314: bool Program::parse_file_location(const std::string&, std::map<int, const char*, std::less<int>, std::allocator<std::pair<const int, const char*> > >&): Assertion `tokens.size() == 5' failed.
Aborted (core dumped)
```

Trying the same number of threads for saxpy_double.ptx gives me an accurate error:

```
$ weft -n 320 saxpy_double.ptx
WEFT ERROR 4: Found max thread count 384 which does not agree with specified count of 320!
WEFT WILL NOW EXIT...
```

but then using the correct number of threads gives me the same error:

```
$ weft -n 384 saxpy_double.ptx
weft: program.cc:314: bool Program::parse_file_location(const std::string&, std::map<int, const char*, std::less<int>, std::allocator<std::pair<const int, const char*> > >&): Assertion `tokens.size() == 5' failed.
Aborted (core dumped)
```

I compiled the weft executable using the instructions on the github page and built the example files using the distributed Makefile. Any advice would be greatly appreciated. Thanks!

**Comment by sharmar** [comment]
What version of CUDA are you using?

**Comment by kellyrowland** [comment]
CUDA 5.0

```
$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2012 NVIDIA Corporation
Built on Fri_Sep_21_17:28:58_PDT_2012
Cuda compilation tools, release 5.0, V0.2.1221
```

**Comment by sharmar** [comment]
Will it be possible for you to upgrade to the latest version of CUDA? The earliest version of CUDA we have tried with WEFT is 5.5.

**Comment by kellyrowland** [comment]
Using CUDA 5.5 seems to have fixed the problem, thank you.

It would probably be a good idea to list CUDA 5.5 as the minimum version as a prerequisite instead of CUDA 5.0.

Thanks again!

**Comment by lightsighter** [comment]
Thanks again for your help!  I pushed an update which does better checking for older versions of PTX and will raise an error for any PTX files older than version 3.2 (which shipped with CUDA 5.5).
